sktime/datatypes/_utilities.py

```diff
@@ -230,14 +287,13 @@ def get_window(obj, window_length=None, lag=None):
     Parameters
     ----------
     obj : sktime compatible time series data container or None
-        if not None, must be of one of the following mtypes:
-            pd.Series, pd.DataFrame, np.ndarray, of Series scitype
-            pd.multiindex, numpy3D, nested_univ, df-list, of Panel scitype
-            pd_multiindex_hier, of Hierarchical scitype
+        if not None, must be of Series, Panel, or Hierarchical scitype
+        all mtypes are supported via conversion to internally supported types
+        to avoid conversions, pass data in one of GET_WINDOW_SUPPORTED_MTYPES
     window_length : int or timedelta, optional, default=-inf
         must be int if obj is int indexed, timedelta if datetime indexed
         length of the window to slice to. Default = window of infinite size
-    lag : int or timedelta, optional, default = 0
+    lag : int or timedelta, optional, default = 0 (correct type zero is inferred)
```

Comment:
default for this argument is `None`, isn't it?

Reply:
I am a user. perhaps I am a dumb one, but 0 is not the same as `None`. As a user I would prefer to see the same default value in the docstring as I see in the function's signature. So, if possible, please, `default = None` as in many other functions in sktime.

Reply:
ok, pushed a suggestion

Reply:
my internal dumb user is happy now:D thanks!